cypress/integration/core-opensearch-dashboards/opensearch-dashboards/apps/data_explorer/discover.spec.js

```diff
@@ -27,17 +27,7 @@ const indexSet = [
 describe('discover app', { scrollBehavior: false }, () => {
   before(() => {
     CURRENT_TENANT.newTenant = 'global';
-    if (Cypress.env('SECURITY_ENABLED')) {
-      /**
-       * Security plugin is using private tenant as default.
-       * So here we'd need to set global tenant as default manually.
-       */
-      cy.changeDefaultTenant({
-        multitenancy_enabled: true,
-        private_tenant_enabled: true,
-        default_tenant: 'global',
-      });
```

Comment:
if we set default tenant as `global` in security system index and not specify tenant for each visit, would that solve the tenant issue? 

is there a case need tenant to be `private`? if there is no such case, i think we can set default tenant as `global` instead `private`

Reply:
For dashboard, no I suppose. And I agree that we should set default tenant as `global` to hide the complicated tenant logic.